web-client/public/js/setup-handlers.js

```diff
@@ -75,9 +91,29 @@ export const setupHandlers = grnState => {
         updateApp(grnState);
     };
 
-    $(GRID_LAYOUT_BUTTON).click(() => setGraphLayout(grnState.graphLayout === FORCE_GRAPH ? GRID_LAYOUT : FORCE_GRAPH));
-    $(FORCE_GRAPH_CLASS).click(() => setGraphLayout(FORCE_GRAPH));
-    $(GRID_LAYOUT_CLASS).click(() => setGraphLayout(GRID_LAYOUT));
+// Grid and Force Graph Layout
```

Comment:
The indentation on _this_ comment looks inconsistent with how the other comments have been re-indented. Pick a convention and stay with it.
```suggestion
    // Grid and Force Graph Layout
```